CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+4.7.0
+---
+* `read-*` functions are now recoverable if the key doesn't exist
```

Comment:
Can you say a bit more here? I don't know precisely what this means/implies. Does it mean that I can wrap a call to `read-string` with `try` if a key doesn't exist, whereas I couldn't before? This ChangeLog entry reads like it was meant for other Pact devs to read it, rather than Pact users.

Reply:
I improved it.